src/mixins/widget.js

```diff
@@ -23,6 +23,10 @@ export const createWidgetMixin = ({ connector } = {}) => ({
       this.factory = connector(this.updateState, () => {});
       this.widget = this.factory(this.widgetParams);
       this.instantSearchInstance.addWidget(this.widget);
+
+      if (this.instantSearchInstance._isSsr) {
```

Comment:
We do want to force render on the server and on the client, but only the first time. With the current implementation a widget that mount after the initial render is calling `__forceRender`.